docs/differences_with_venv.rst

```diff
@@ -0,0 +1,17 @@
+Differences with venv
+=====================
+
+- ``venv`` is bundled with Python itself (starting from 3.3+), ``virtualenv``
+  is a third party package. This allows for ``virtualenv`` to have greater
+  flexibility when dealing with several Python versions.
```

Comment:
I don't think the flexibility comes from the fact that it's bundled.

Reply:
I don't understand, it's not bundled, that's why it has greater flexibility?

I also deduced this from your resentation, given there are multiple installation options and support for any Python version.